### PR TITLE
Update seed info

### PR DESCRIPTION
**Pull request by JSENL**
Updated seed info and added <Link> to name and picture

### CODE REVIEW

File: src/components/ListPage.js [hunk 1]
```diff
@@ -8,10 +8,14 @@ const ListPage = props => {
...
       <div className="small-12 medium-6 columns about-pets">
         <div className="about-pets-avatar">
+        <Link to={`/pets/${id}`}>
           <img className="avatar-image" src={img_url} alt={name} />
+          </Link>
         </div>
         <div className="about-pets-author">
+        <Link to={`/pets/${id}`}>
           <p className="author-name">{name}</p>
+          </Link>
```
Comment (prophetsfall) [inline src/components/ListPage.js]: This indent level should be back one to line up with the `<Link` above.

File: src/components/ListPage.js [hunk 2]
```diff
@@ -8,10 +8,14 @@ const ListPage = props => {
     <div className="row add-pets-section">
       <div className="small-12 medium-6 columns about-pets">
         <div className="about-pets-avatar">
+        <Link to={`/pets/${id}`}>
           <img className="avatar-image" src={img_url} alt={name} />
+          </Link>
```
Comment (prophetsfall) [inline src/components/ListPage.js]: This link should also be backed up.

Comment (JSENL) [reply src/components/ListPage.js]: So you want the  and <Link> to be spaced the same or that you want  to be more indented?

Comment (JSENL) [reply src/components/ListPage.js]: OK updated https://github.com/xshigeox/adopt_a_pet.git

File: src/components/ListPage.js [hunk 3]
```diff
@@ -2,32 +2,36 @@ import React from "react"
...
-      <div className="small-12 medium-6 columns add-friend">
-        <div className="add-friend-action">
-          <Link to={`/pets/${id}`}>
-            <button className="button primary small">Adopt Me</button>
-          </Link>
-        </div>
-      </div>
-    </div>
-  )
+	return (
```
Comment (prophetsfall) [inline src/components/ListPage.js]: You went in a little too far. 
I would suggest backing up the code on line 5 to be even with the line above it, but put a line break between the two. 
Then you can line the return up with that line. From there you would keep the first div even with the return, and indent each line appropriately from there. If you need a hand with that check with your peers, or put a question in the queue.

**Comment by JSENL** [comment]
Hey this has been updated, can you please review?

**Pull request merged**